rdrf/rdrf/templates/two_factor/core/login.html

```diff
@@ -2,6 +2,16 @@
 
 {% load i18n static two_factor %}
 
+{% block extrahead %}
+<script>
+function isIE() {
+  return (window.navigator.userAgent.indexOf('MSIE ') > 0 || window.navigator.userAgent.indexOf('Trident/') > 0);
+}
+
+if (isIE()) alert("{% trans "TRRF does not support Internet Explorer. Please use another browser" %}");
```

Comment:
This is probably fine, but I'm just wondering if the end user for (e.g.) MND would understand what "TRRF" is. Might be better to be a bit more generic and say "This site does not support..."